README.md

```diff
@@ -32,6 +32,10 @@ Question keys
     * `"on"` service data key name to use for comparison (e.g. "lot" for lots)
     * `being` a list of acceptable values for the key. If service data key value matches one of the values in the `being` the
        question is kept, otherwise the question is removed from the section for the given service
+* `limits` a dictionary of validation/type requirements to limit the possible question values. Available fields:
+    * `min_value` integer, sets the minimum value for percentage questions
+    * `max_value` integer, sets the maximum value for percentage questions
+    * `integer_only` boolean, when `true` makes the percentage question accept only integer values instead of numbers
```

Comment:
`limits` seems really generic but then the description for `min_value` seems really specific. I'm unsure whether the intention is to use `limits` for other types as well (in which case `min_value` seems a bit vague) or just for the percentage type (in which case `limits` seems a bit vague).